scripts/apps/vocabularies/constants.ts

```diff
@@ -0,0 +1,18 @@
+export const MEDIA_TYPES = {
+    GALLERY: {
+        id: 'media',
+        label: 'Media gallery'
```

Comment:
Sorry, forgot this one. You said you are already translating this in the templates, which is fine, but it won't get picked up to be sent for translation by the grunt script if you don't wrap it in `gettext`

Reply:
done